lib/Server.js

```diff
@@ -32,6 +31,7 @@ const createDomain = require('./utils/createDomain');
 const runBonjour = require('./utils/runBonjour');
 const routes = require('./utils/routes');
 const schema = require('./options.json');
+const SockjsServer = require('./utils/server/sockjsServer');
 
 // Workaround for sockjs@~0.3.19
```

Comment:
Let's move workaround to sockjsServer

Reply:
You mean this workaround? https://github.com/webpack/webpack-dev-server/blob/master/lib/Server.js#L41

Reply:
Yes